src/azure-cli/azure/cli/command_modules/storage/util.py

```diff
@@ -16,6 +16,19 @@ def collect_blobs(blob_service, container, pattern=None):
     return [name for (name, _) in collect_blob_objects(blob_service, container, pattern)]
 
 
+def collect_blobs_track2(blob_service, container, pattern=None):
+    """
+    List the blobs in the given blob container, filter the blob by comparing their path to the given pattern.
+    Return files only.
+    """
+    blobs = collect_blob_objects(blob_service, container, pattern)
+    r = []
+    for name, blob in blobs:
+        if blob.content_settings.content_md5 is not None:
```

Comment:
Why adding such logic for track2?

Reply:
This logic is used to exclude directory as copy directory will raise an error. It seems like not all file has `content_md5`, I'll use try-except instead.